lms/djangoapps/teams/csv.py

```diff
@@ -19,3 +26,234 @@ def load_team_membership_csv(course, response):
         "Team membership CSV download is not yet implemented."
     )
     response.write(not_implemented_message + "\n")
+
+
+class TeamMembershipImportManager(object):
+    """
+    A manager class that is responsible the import process of csv file including validation and creation of
+    team_courseteam and teams_courseteammembership objects.
+    """
+
+    def __init__(self, course):
+        self.validation_errors = []
+        self.teamset_ids = []
+        self.user_ids_by_teamset_id = {}
+        self.teamset_ids = []
+        self.number_of_record_added = 0
+        self.course = course
+        self.max_errors = 0
+        self.existing_course_team_memberships = {}
+        self.existing_course_teams = {}
```

Comment:
Couple nits: 
* `teamset_ids` is declared twice.
* `number_of_record_added` should be `number_of_records_added`.